### PR TITLE
issue/48 Add documentation for migrating to rub from adapt-cli

### LINKED ISSUE #48: Add documentation for the pathways from adapt-cli to a builds folder
Please add a section to the readme regarding the pathways from adapt-cli to a builds folder (i.e. using rub-cli instead of grunt).  This should cover:

- Manually creating the 'builds' folder.
- After running `grunt build`, moving the new course into the 'builds' folder.
- Deleting the 'course' folder in 'src'.  Otherwise, the 'build' folder will keep being recreated when rub is ran.

Note that this does not apply when using adapt-setup instead of adapt create course.  However, it should still be documented.

Thanks.

### PR DESCRIPTION
**Pull request by swashbuck**
Resolves #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,3 +100,12 @@ These three commands (``tracking:remove``, ``tracking:insert`` and ``tracking:re
 ### Flags vs commands vs options
 Commands will typically run at the exclusion of all other non-dependent tasks, such that ``rub prettify`` will only perform a json prettification task. Flags now assume a build should happen, such that ``rub -P`` will build and prettify, ``rub -Pz`` will build, prettify and zip, etc. Options specify a value, such as `rub -p=10`, to allow 10 parallel tasks.
 
+### Migrating from adapt-cli to rub
+In order to use rub, your Adapt project's folder structure will require a few changes.
+
+1. Create a `builds` folder in the root of your project.
+2. Run `grunt build` to create your initial course. This creates a course folder called `build.`
+3. Move the `build` folder into the `builds` folder and rename it using your course name (ex. `p101`).
+4. Finally, delete the `course` folder that lives in `src`. Otherwise, the `build` folder will be recreated each time you run rub.
```
Comment (oliverfoster) [inline README.md:109]:
```suggestion
2. Create a folder inside the `builds` folder using your module name (e.g. `p101)
3. Move/remove the `src/course` folder into the `builds/p101` folder 
4. Run `rub dev p101` to build
```

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -100,3 +100,12 @@ These three commands (``tracking:remove``, ``tracking:insert`` and ``tracking:re
 ### Flags vs commands vs options
 Commands will typically run at the exclusion of all other non-dependent tasks, such that ``rub prettify`` will only perform a json prettification task. Flags now assume a build should happen, such that ``rub -P`` will build and prettify, ``rub -Pz`` will build, prettify and zip, etc. Options specify a value, such as `rub -p=10`, to allow 10 parallel tasks.
 
+### Migrating from adapt-cli to rub
+In order to use rub, your Adapt project's folder structure will require a few changes.
+
+1. Create a `builds` folder in the root of your project.
+2. Create a folder inside the `builds` folder using your module name (e.g. `p101)
+3. Move/remove the `src/course` folder into the `builds/p101` folder
```
[the remark this answers is not in the document]
Comment (oliverfoster) [reply README.md:108]: Yup, that works.

[the remark this answers is not in the document]
Comment (oliverfoster) [reply README.md:108]: CLean up as you wish, my comment was more about removing `grunt` from the entire process.

**Comment by swashbuck** [comment]
@oliverfoster I've revised the text a bit.  Let me know if this is approved and can be merged.

File: README.md [hunk 3]
```diff
@@ -100,3 +100,10 @@ These three commands (``tracking:remove``, ``tracking:insert`` and ``tracking:re
 ### Flags vs commands vs options
 Commands will typically run at the exclusion of all other non-dependent tasks, such that ``rub prettify`` will only perform a json prettification task. Flags now assume a build should happen, such that ``rub -P`` will build and prettify, ``rub -Pz`` will build, prettify and zip, etc. Options specify a value, such as `rub -p=10`, to allow 10 parallel tasks.
 
+### Migrating from adapt-cli to rub
+In order to use rub, your Adapt project's folder structure will require a few changes.
+
+1. Create a `builds` folder in the root of your project.
+2. Create a folder inside the `builds` folder using your module name (e.g. `p101).
+3. Move the contents of the 'src/course' folder into 'builds/p101'. Then, remove 'src/course' entirely.
```
Comment (oliverfoster) [inline README.md:108]:
```suggestion
3. Move the contents of the 'src/course' folder into 'builds/p101/course'. Then, remove 'src/course' entirely.
```

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**